README.md

```diff
@@ -26,6 +26,7 @@ The extensions herein are used in the [Phase Two](https://phasetwo.io) cloud off
     - [Resources](#resources)
     - [Mappers](#mappers)
     - [Admin Events](#admin-events)
+    - [User Events](#user-events)
```

Comment:
This is getting big for the README. Can you create a page `docs/events.md` and put both the admin and user events in that page? Then you can link from the section below.